README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
It would improve clarity to specify what 'task install' installs. Mention the purpose of the dependencies being installed to give users context.